guides/doc-Configuring_Load_Balancer/master.adoc

```diff
@@ -34,7 +34,7 @@ include::common/modules/proc_configuring-remaining-smart-proxy-servers-with-defa
 
 include::common/modules/con_configuring-smart-proxy-server-with-custom-ssl-certificates-for-load-balancing-without-puppet.adoc[leveloffset=+2]
 
-include::common/modules/proc_creating-custom-ssl-certificates-for-smart-proxy-server.adoc[leveloffset=+3]
+include::common/modules/proc_creating-a-custom-ssl-certificate.adoc[leveloffset=+3]
```

Comment:
```suggestion
:ProductName: {SmartProxyServer}
include::common/modules/proc_creating-a-custom-ssl-certificate.adoc[leveloffset=+3]
```
because
```
asciidoctor: WARNING: skipping reference to missing attribute: productname
```